CHANGELOG.md

```diff
@@ -52,13 +52,17 @@ v1.9.0 [unreleased]
 -	[#21358](https://github.com/influxdata/influxdb/pull/21358): fix: disable MergeFiltersRule until it is more stable
 -	[#21516](https://github.com/influxdata/influxdb/pull/21516): fix: FGA enablement
 
-v1.8.6 [unreleased]
+v1.8.6 [2021-05-21]
 -------------------
 
+Note: No OSS specific changes were made in this release.
```

Comment:
add -> This release was created to support InfluxDB Enterprise 1.8.6.

Reply:
Added.